src/App.tsx

```diff
@@ -1,17 +1,166 @@
-import React from 'react';
+import React, { useState } from 'react';
+
 import './App.css';
 
+import { PreparedTodoType } from './types/PreparedTodoType';
+
 import users from './api/users';
+import todos from './api/todos';
+
+import { TodoList } from './components/TodoList';
+import { UserType } from './types/UserType';
+
+const preparedTodos: PreparedTodoType[] = todos.map(todo => ({
+  ...todo,
+  user: users.find(user => user.id === todo.userId) || null,
+}));
 
 const App: React.FC = () => {
+  const [todoList, addTodo] = useState<PreparedTodoType[]>(preparedTodos);
+
+  const [isCompl, setIsCompl] = useState<boolean>(false);
+
+  const [title, setTitle] = useState<string>('');
+
+  const [selectedUserId, setUserId] = useState<number>(-1);
+
+  const [titleErr, setTitleErr] = useState(false);
+
+  const [userErr, setUserErr] = useState(false);
```

Comment:
Use full variable names
```suggestion
  const [isCompleted, setIsCompl] = useState<boolean>(false);

  const [title, setTitle] = useState<string>('');

  const [selectedUserId, setUserId] = useState<number>(-1);

  const [titleError, setTitleErr] = useState(false);

  const [userError, setUserErr] = useState(false);
```